src/gf-oemid

```diff
@@ -18,6 +18,11 @@ src="$1"
 dest="$2"
 oemid="$3"
 
+if [[ $src != *.qcow2 ]]; then
```

Comment:
Maybe change this to run `qemu-img info $src >/dev/null` ?  We may want to e.g. support using sparse files or VMDK or something else in the future.

Reply:
Ah cool...I was wondering about supporting other formats.

Reply:
Hmm, `qemu-img info` exits cleanly for `.gz` files too.  I guess I could change the check to be

```
if [[ $src == *.gz ]]; then
...
```

Reply:
Oh, I see, it thinks anything it can't detect is just raw.

Yeah...let's explicitly error out for `*.gz`!